packages/x-date-pickers/src/models/fields.ts

```diff
@@ -117,6 +118,17 @@ export interface FieldRef<TSection extends FieldSection> {
 
 export type FieldSelectedSections = number | FieldSectionType | null | 'all';
 
+export interface FieldOwnerState extends PickerOwnerState {
```

Comment:
In v9, I should be able to access a lot more info if needed, but for now the structure of the component makes it very hard to put anything useful here.

I added the `disabled` and `readOnly` props because when used as standalone then are needed since `isPickerDisabled` and `isPickerReadOnly` are not reliable.

I think that's a good start that we can enrich in the future :ok_hand: 
This object will also be used for the `inputAdornment`, `openPickerButton` and `openPickerIcon` slots once those are handled by the field component.